app/api/migrations/migrations/19-rename-uploads-to-files/index.js

```diff
@@ -8,6 +8,10 @@ export default {
   async up(db) {
     process.stdout.write(`${this.name}...\r\n`);
 
+    try {
+      await db.collection('files').drop();
```

Comment:
This is not idempotent, which has been an 'unspoken' rule of migrations.  Not sure if it merits here to go the extra effort to ensure that a second run will not forever delete the already-migrated files?

Reply:
@RafaPolit changed it so it only runs the migration when has not ran before, this assumes that if uploads collection does not exist then the migration has already ran.